### DISCUSSION TITLE
コメント欄のLGTMボタンと数字の間隔が狭い

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#222] [要望・アイデア・不具合]
コメント欄のLGTMボタンと、その右に表示されている数字の間隔がほとんど無いように見えます。

<img width="353" alt="Screen Shot 2022-05-14 at 7 26 47" src="https://user-images.githubusercontent.com/1148320/168397057-16294c32-6ee2-4fa3-9566-f24da2c7eae5.png">

多少間を空けた方が良いと思うのですがいかがでしょうか？

<img width="218" alt="Screen Shot 2022-05-14 at 7 29 20" src="https://user-images.githubusercontent.com/1148320/168397140-1315adc3-3fa5-4dc0-89ba-487b74241523.png">

上記の例は[こちらのコメント](https://qiita.com/kubochiro/items/557ad6276b106ab9180b#comment-a86c1885aa1376e84d9f)のスクショです。

**Comment by tanaka-asami415** [comment]
@JunichiIto 
サンプル画像もつけてくださり、ありがとうございます！🙏
確かに間があった方がよさそうです！
こちら先日いただいた、画像と背景の境目のissueと合わせて対応いたします:thumbsup:

**Reply by JunichiIto** [reply]
ありがとうございます。よろしくお願いします。

**Reply by kyntk** [reply]
@JunichiIto 
大変お待たせしました！
LGTMボタンと数字との間に隙間ができるように変更しました！
![](https://user-images.githubusercontent.com/17251829/171550327-2493b7bd-e83a-41aa-9cd6-da451f397841.png)

**Reply by JunichiIto** [reply]
確認しました。いい感じにマージンが空きましたね。
ご対応どうもありがとうございます！
